ResourcesCard.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect } from "react";
+
+import { sendEvt } from "services/event.service";
+
+import { CalloutCard } from "component/cards/CalloutCard";
+
+import libraryIcon from "./library.png";
+import imgFallback from "img/resource_fallback.png";
+
+import styles from "./ResourcesCard.module.scss";
+
+const ResourcesCard = () => {
+  const [resources, setResources] = useState([]);
+  const [primaryResource, setPrimaryResource] = useState(none);
+  const [secondaryResources, setSecondaryResources] = useState([]);
+
+  var sendEventTracking = (resourceUuid) => {
+    const evt = new Object();
+    evt = {
+      event_name: "web_dashboard_card_" + r.id + "_" + r.title + "_tap"
+    };
+    sendEvt(evt);
+    console.log("sending event");
+  };
+
+  // TODO
+  useEffect(async () => {
+    let req = await getResources(userId);
+    setResources(req.resources);
+
+    let primaryResource = resources[0];
+    let secondaryResources = resources.slice(1, 5);
+    setPrimaryResource(primaryResource);
+    setSecondaryResources(secondaryResources);
+    return;
+  });
+
+  // Get image of first resource
+  const i = () => {
+    const image = primaryResource.icon;
+    if (!primaryResource.icon || primaryResource.icon === undefined) {
+      return imgFallback;
+    }
+
+    if (image === true) {
+      return image;
+    }
+  };
+
+  return (
+    <div class={styles.ResourcesCard}>
+      <div className="content">
+        <h1>{title}</h1>
+        <br />
```

Comment:
Instead of using break elements to add visual space on the page, it would be preferable to just add styling to the title element to add the space (probably using `margin-bottom`